.coderabbit.yaml

```
@@ -0,0 +1,90 @@
+    - "*.html"
+    - "*.md"
+    - "package.json"
+    - ".github/**"
+
+# Language-specific settings
+language_settings:
+  javascript:
+    # Enable additional JavaScript/TypeScript specific checks
+    enable_eslint: true
+    enable_prettier: true
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)